public/js/main.js

```diff
@@ -308,6 +308,14 @@ BlocksApp.filter('timeDuration', function() {
     return getDifficulty(hashes);
   };
 }) 
+.filter('bigNumber', function() {
+    return function(val) {        
+        if (val == null) return val;
+        var parts = val.toString().split(".");
+        parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+        return parts.join(".");
+    };
+  }) 

```

Comment:
Is there a built-in directive we can use? Like this one: https://docs.angularjs.org/api/ng/filter/number

Reply:
Can we adjust that? "Number of decimal places to round the number to. If this is not provided then the fraction size is computed from the current locale's number formatting pattern. In the case of the default locale, it will be 3."

Or use [`currency`](https://docs.angularjs.org/api/ng/filter/currency)?

Otherwise we lose the locale customization of `,` v.s `.`.

Reply:
we can adjust (didn't work for me) but how many decimal places we should have after .?
Currency round up to 2 digits by default and add currency sign.

I think it's easier to add locale customization - , or . if it make sense (decimal and thousands separators)

Reply:
Yeah I guess we'd have to tell it to display max decimals, then we're back to truncating trailing zeros.